portals/publisher/package.json

```diff
@@ -31,15 +31,18 @@
         "@apidevtools/swagger-parser": "^10.0.1",
         "@asyncapi/parser": "1.3.2",
         "@asyncapi/react-component": "^0.19.0",
+        "@autocomplete/material-ui": "0.0.17",
         "@hapi/hapi": "^18.4.1",
         "@material-ui/core": "^4.11.3",
         "@material-ui/icons": "^4.9.1",
+        "@material-ui/lab": "^4.0.0-alpha.60",
```

Comment:
Can;t find a usage of UI lab in the code, Shall we avoid using MUI lab , If it's must to use a component from lab, copy it to our codebase and use it

Reply:
Autocomplete component coming from that. https://github.com/wso2/apim-apps/pull/144/files#diff-2151296585a6babc62925bf1a6be90d840a2031ab720edc69021753cd335bd70R30

Reply:
Shall we copy the code into our code base
example:
wso2/apim-apps/portals/publisher/source/src/app/components/Shared/Alert.jsx

Reply:
because lab package is an alpha package, pls try to avoid using that

Reply:
https://github.com/mui/material-ui/blob/v4.x/packages/material-ui-lab/src/Autocomplete/Autocomplete.js
have to copy

- `useAutocomplete` file
- For internal svg icon references let's use a matching [mui icon](https://v4.mui.com/components/material-icons/#material-icons)

Reply:
Added the custom code from https://github.com/wso2/apim-apps/pull/144/commits/466220bf27f6bb46343cc3d3c5b36be8dc9d8d45. Its better to remove it when it has a stable release